### DISCUSSION TITLE
Reown appkit flutter does nothing in build release

### DISCUSSION THREAD

**Discussion by Sleechigo** [reown-com/appkit#3149] [Q&A]
Hi, in my debug reown appkit works fine, but after I build a release, the connect button doesn't work and is just dormant after clicking.

**Comment by rtomas** [comment]
cc @quetool

**Comment by quetool** [comment]
Hello @Sleechigo ! Please open an issue in flutter repo https://github.com/reown-com/reown_flutter with your current flutter version, your build command and a minimal reproducible code if possible! Thanks!
